pyonepassword/_op_cli_config.py

```diff
@@ -71,11 +71,12 @@ def __init__(self, configpath=None):
             account_map[account.shorthand] = account
         self.account_map = account_map
 
-    def _get_config_path(self):
-        configpath = None
+    def _get_config_path(self) -> pathlib.Path:
+        configpath: pathlib.Path = None
```

Comment:
I'm surprised mypy doesn't complain that you're assigning None to something that isn't nullable.

Reply:
I relaxed some of mypy's `None` complaints with

```
implicit_optional = True
strict_optional = False
```

So probably something to do with that.

Reply:
right, yeap, that'll do it